packages/core/upload/admin/src/utils/index.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable check-file/no-index */
```

Comment:
I'm confused why you made this change. As [I mentioned](https://github.com/strapi/strapi/pull/21896#discussion_r1827510203), this is not a solution. The problem is still present in many other files. No point silencing it in just one file.